tests/tests_fabric/run_standalone_tasks.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# Copyright The Lightning AI team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# THIS FILE ASSUMES IT IS RUN INSIDE THE tests/tests_fabric DIRECTORY
+export PYTHONPATH="${PYTHONPATH}:$(pwd)"
+export PYTHONPATH="${PYTHONPATH}:$(pwd)/.."
+
+MAX_RETRIES=3
+
+retry_command() {
```

Comment:
This improvement could be added directly to the standalone script, where we assign a retry flag to specific standalone tests either by updating a list in the script or by checking whether the test has a specific decorator

Reply:
If it complicates things, I would rather go for this simple way. I prefer not to spend too much time on this.

Reply:
I'm proposing this because there are other standalone tests that could use a retry mechanism. It would be an useful improvement